dev-cpp/amqp-cpp/amqp-cpp-4.3.16.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+MY_PN=${PN^^}
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="AMQP-CPP is a C++ library for communicating with a RabbitMQ message broker"
+HOMEPAGE="https://github.com/CopernicaMarketingSoftware/AMQP-CPP"
+SRC_URI="https://github.com/CopernicaMarketingSoftware/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
```

Comment:
Wrap long lines? Even if you can't fit the URL, it would be better to fit the rest on a single line.

Reply:
I find it easier to comprehend URL -> filename parings when the are on the same line.

Reply:
In any case, it's a style tweak, and I'm not inclined to make further of those in this PR as it's gone on a while. But I agree wit hfloppym.